i18n/ja/docusaurus-plugin-content-docs/current/guides/mobile-plugin-ima-ios.md

```diff
@@ -0,0 +1,51 @@
+---
+title: UID2 IMA Plugin for iOS
+description: A guide for publishers using IMA for ad requests on iOS apps.
+hide_table_of_contents: false
+sidebar_position: 15
+---
+
+# UID2 IMA Plugin for iOS Integration Guide
+
+The UID2 Interactive Media Ads (IMA) Plugin for iOS enables publishers that use the [Google Interactive Media Ads SDK](https://developers.google.com/interactive-media-ads/docs/sdks/html5/client-side/) to send [UID2 tokens](../ref-info/glossary-uid.md#gl-uid2-token) as [secure signals](https://support.google.com/admob/answer/11556288?hl=en-GB) in ad requests from iOS apps. This is done automatically, with no direct coding needed from the app developer.
```

Comment:
The Google IMA SDK link is wrong.  It points to the HTML5 SDK when it should be pointing to the iOS SDK -->

https://developers.google.com/interactive-media-ads/docs/sdks/ios/client-side

Reply:
Fixed, thx.

Reply:
It's still showing as incorrect.... has the commit been pushed?

<img width="1492" alt="Screenshot 2023-08-18 at 3 09 19 PM" src="https://github.com/IABTechLab/uid2docs/assets/989928/aa41c870-68a2-44d3-8b3b-e938673e9a7b">

Reply:
I hadn't realized it occurs twice in the doc... fixed the other instance,  not this. It's fixed now.  Thanks for the spot.